.github/dependabot.yml

```diff
@@ -7,6 +7,8 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    target-branch: "main"
     allow:
-      - dependency-type: "development"
+      - dependency-name: "@metamask/*"
+        dependency-type: "development"
```

Comment:
I'm curious why this is here. I would think we'd want _all_ updates to our packages, not just those that are `devDependencies`. :thinking: